assets/subworkflows/espresso/band_structure_average_esp.yml

```diff
@@ -14,19 +16,31 @@ units:
     type: assignment
 -   config:
         execName: pw.x
-        flavorName: pw_scf
+        flavorName: pw_scf_multimaterial
```

Comment:
We might not need this per https://github.com/Exabyte-io/exabyte-application-flavors/pull/58/files#r1014539920